### PR TITLE
adds contributors

### PR DESCRIPTION
**Pull request by argonlaser**
https://github.com/microplan-xyz/microplan/issues/78

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -203,6 +203,13 @@ $ npm version major # for deprecation
 
 **Important:** `microplan` uses an automated publishing system. That means whenever your PR is merged, `microplan` will be published automatically to npm (based on the version specified in the package.json)
 
+## Contributors
+Authored and maintained by [@scriptnull](https://github.com/scriptnull) and [@argonlaser](https://github.com/argonlaser) with help from [contributors].
+
```
Comment (argonlaser) [inline README.md]: Need we add a twitter handle here?

Comment (scriptnull) [reply README.md]: let's just have GitHub handles!

File: README.md [hunk 2]
```diff
@@ -203,6 +203,13 @@ $ npm version major # for deprecation
...
+[@scriptnull]: (https://github.com/scriptnull)
```
Comment (scriptnull) [inline README.md]: let's remove this line as line 207 already has the link in () braces.

File: README.md [hunk 3]
```diff
@@ -203,6 +203,13 @@ $ npm version major # for deprecation
...
+[@argonlaser]: (https://github.com/argonlaser)
```
Comment (scriptnull) [inline README.md]: let's remove this line as line 207 already has the link in () braces.

File: README.md [hunk 4]
```diff
@@ -203,6 +203,13 @@ $ npm version major # for deprecation
...
+[contributors]: http://github.com/microplan-xyz/microplan/contributors
```
Comment (scriptnull) [inline README.md]: Let's remove this line and move the URL to line number 207 just like the usernames.

**Pull request merged**